docs/ref/clvm.md

```diff
@@ -125,24 +125,23 @@ And in this example, an atom that was read in as an integer is appended to a str
 
 ### Atoms as Byte Arrays
 
-The atom is treated as an array of bytes, with a length. No specific semantics are assumed, except as specified in the instruction.
+The atom is an array of bytes, with a length. No specific semantics are assumed, except as specified in the instruction.
 
-### Unsigned Integer
+### Signed Integer
 
-An unsigned integer of arbitrary length. If more bits are required to perform an operation with atoms of different length, the atom is virtually extended with zero bytes to the left.
```

Comment:
> If more bits are required to perform an operation with atoms of different length, the atom is virtually extended with zero bytes to the left

This is typically referred to as sign-extension, and it also applies to negative numbers (but then extending with `0xff`.

sign-extension happens as an implementation detail in some places where we convert atoms into machine words (`i32`). But this doesn't affect the abstraction of how atoms are interpreted as integers.

The only operator, I can think of, where it might make sense to talk about sign extensions is `ash`, which already clarifies this behavior.